app/Models/CategoryModel.php

```diff
@@ -87,17 +91,17 @@ public function findAllNested(array $categoryIds): array
             ->parents()
             ->whereIn("{$this->table}.id", $categoryIds)
             ->select(implode(', ', $selects))
-            ->join('threads', 'threads.id = categories.last_thread_id')
-            ->join('users', 'users.id = threads.author_id')
+            ->join('threads', 'threads.id = categories.last_thread_id', 'LEFT')
```

Comment:
Where are these used? If it's only for the category list in the sidebar then having only categories with posts in them is desirable so that users don't waste time clicking into categories that don't have any content. 

If it's used elsewhere, then I agree with this.

Reply:
This is very important :smile: 
You will not be able to create a new topic in an empty Thread. Select menu is empty.
Everything works for you because you are using a Seeder or an already filled database

Reply:
Ok, that's what I was wanting to verify. :) The rest looks good. Approved.